### PR TITLE
 translated from English to French + several ortography corrections

### PR DESCRIPTION
**Pull request by AurelBouchet**

### CODE REVIEW

File: src/components/Map/MarkersComponent.jsx [hunk 1]
```diff
@@ -47,9 +47,9 @@ const StationsMarkers = ({
       <Popup>
         {station.fields.address}
         <br />
-        {station.fields.available_bikes} bikes available
+        {station.fields.available_bikes} bikes availables
```
Comment (teasmade) [inline src/components/Map/MarkersComponent.jsx]: still English?!

Comment (AurelBouchet) [reply src/components/Map/MarkersComponent.jsx]: ah oui  je change

**Review by teasmade — APPROVED** [review]
It's bon for moi ;)

**Review by bastienapp — APPROVED** [review]
Parfait 🇬🇧 -> 🇫🇷 👍

**Pull request merged**